openapi.yaml

```diff
@@ -120,7 +120,7 @@ components:
         - FederatedClaims
         - ServiceAccount
 
-    RbacCollection:
+    Rbac:
```

Comment:
You should change line 20 as well to `$ref: '#/components/schemas/RbacCollection'`